public_html/wp-content/mu-plugins/blocks/source/blocks/organizers/block-controls.js

```diff
@@ -61,28 +61,34 @@ export class BlockControls extends Component {
 			default :
 				output = (
 					<Placeholder
-						className={ classnames( 'wordcamp-block-edit-placeholder', 'wordcamp-block-edit-placeholder-no-mode' ) }
+						className={ classnames( 'wordcamp__edit-placeholder', 'has-no-mode' ) }
 						icon={ icon }
 						label={ LABEL }
 					>
-						<div className="wordcamp-block-edit-mode-option">
-							<Button
-								isDefault
-								isLarge
-								onClick={ () => {
-									setAttributes( { mode: 'all' } );
-								} }
-							>
-								{ getOptionLabel( 'all', options.mode ) }
-							</Button>
-						</div>
+						<EditModeOption
```

Comment:
`MUST` - I thought we were removing the elements for these? It looks like they one of the commits in this PR removed them, but a later one added them back.

Reply:
Where are you seeing this? Its already removed, I force pushed the branch today, so you might need to force pull it if you are testing on local

Reply:
Resolving this, please re-open if action is needed here

Reply:
Huh, that's weird, I'm not seeing `EditModeOption` in `production` either, and it's not shown as being removed [in the current diff](https://github.com/WordPress/wordcamp.org/pull/123/files). I'm confused.

Reply:
Noting here that confusion was because `EditModeOption` was never merged, so it was not showing as removed.